src/OutputImageParam.h

```diff
@@ -126,18 +126,6 @@ class OutputImageParam {
      * inspecting its min, extent, or stride. */
     EXPORT const Dimension dim(int i) const;
 
-    /** Get or constrain the shape of the dimensions. Soon to be
-     * deprecated. Do not use. */
-    // @{
-    OutputImageParam set_min(int i, Expr e) {dim(i).set_min(e); return *this;}
```

Comment:
I think we should keep these with an explicit attribute deprecated (instead of just a comment) for a while. This PR is going to break a lot of code otherwise.